src/components/PaginationContainer.ts

```diff
@@ -0,0 +1,150 @@
+import { Component, ReactElement, createElement } from "react";
+import { findDOMNode } from "react-dom";
+import * as dijitRegistry from "dijit/registry";
+import * as classNames from "classnames";
+import * as dojoLang from "dojo/_base/lang";
```

Comment:
Not required since the birth of lexical scoping.

Reply:
removed dojo lang